Library/Formula/pius.rb

```diff
@@ -10,7 +11,7 @@ def install
     ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
     # Replace hardcoded gpg path (WONTFIX):
     # https://sourceforge.net/p/pgpius/bugs/12/
-    inreplace "libpius/constants.py", "/usr/bin/gpg", "#{HOMEBREW_PREFIX}/bin/gpg"
+    inreplace "libpius/constants.py", %r{/usr/bin/gpg2?}, "#{HOMEBREW_PREFIX}/bin/gpg"
```

Comment:
Given the package actually depends on `gpg` and not `gpg2`, shouldn't this always point to the `gpg` binary?  Currently pius works with both `gpg` and `gpg2`.  The next release will be `gpg2` only at which time I'd say change the depends and just update the regex.

All that happened is that the author changed the default.  The correct binary can always be given via CLI option as well.